packages/@aws-cdk/cloud-assembly-schema/README.md

```diff
@@ -0,0 +1,62 @@
+## Cloud Assembly Schema
+<!--BEGIN STABILITY BANNER-->
+
+---
+
+![Stability: Stable](https://img.shields.io/badge/stability-Stable-success.svg?style=for-the-badge)
+
+
+---
+<!--END STABILITY BANNER-->
+
+This module is part of the [AWS Cloud Development Kit](https://github.com/aws/aws-cdk) project.
+
+## Cloud Assembly
```

Comment:
In a sense this supersedes the unmaintained **cloud assembly spec** we have under [design/cloud-assembly](https://github.com/aws/aws-cdk/blob/master/design/cloud-assembly.md).

I agree with @nija-at that this requires more detail, but in order to move this forward, here's my suggestion:

1. Delete `design/cloud-assembly.md` as part of this PR (instead of actually deleting I would replace it with a file that points to this file).
2. Follow up with a subsequent PR to expand this README a bit and make sure it becomes the official spec.